test/services/authorization-codes.test.js

```diff
@@ -2,23 +2,15 @@ let mock = require('mock-require');
 mock('redis', require('fakeredis'));
 
 let should = require('should');
-let _ = require('lodash');
 let services = require('../../lib/services');
 let authCodeService = services.authorizationCode;
 let db = require('../../lib/db')();
 
 describe('Authorization Code Tests', function () {
   let newCode, codeFromDb;
 
-  before(function (done) {
-    db.flushdbAsync()
-    .then(function (didSucceed) {
-      if (!didSucceed) {
-        console.log('Failed to flush the database');
-      }
-      done();
-    })
-    .catch(done);
+  before(function () {
+    return db.flushdbAsync();
```

Comment:
👍 👍 👍 

You can also do

`before(db.flushdbAsync)`

Reply:
👍

Reply:
actually it doesn't work that way

Reply:
No? That's weird 🤔